common/scala/build.gradle

```diff
@@ -13,6 +13,8 @@ dependencies {
 
     compile 'io.spray:spray-json_2.11:1.3.3'
 
+    compile 'com.github.pureconfig:pureconfig_2.11:0.7.2'
```

Comment:
Mozilla Public License 2.0. @csantanapr do we need to do anything for Apache on new licenses we're depending on?

Reply:
https://mvnrepository.com/artifact/com.github.pureconfig/pureconfig_2.11/0.7.2

Reply:
It should be OK MPL 2.0 is category 2
https://www.apache.org/legal/resolved.html#category-b
we don't include the source, we just need the binary